### PR TITLE
[Viewing Room] Add tests

### PR DESCRIPTION
**Pull request by damassi**
This adds some tests to the Viewing Room app. 

(Assigning again to you @dzucconi just so you can see some of our testing patterns.)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -49,8 +47,17 @@ module.exports = {
...
     "@typescript-eslint/no-var-requires": 0,
+    "react/display-name": 0,
+    "react/prop-types": 0,
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
+    "sort-imports": [
+      "warn",
+      {
+        // Unfortunately there's no autofixer for this
+        ignoreDeclarationSort: true,
```
Comment (damassi) [inline .eslintrc.js:58]: In which case we lean on the autofixer to sort named export keys on individual imports, but don't worry about alpha for the full import list as thats a real pain.

File: src/Apps/ViewingRoom/Routes/Statement/__tests__/ViewingRoomStatementRoute.test.tsx [hunk 2]
```diff
@@ -1,5 +1,130 @@
-describe.skip("ViewingRoomStatementRoute", () => {
-  it("todo", () => {
-    //
+import React from "react"
+import { MockBoot, renderRelayTree } from "DevTools"
+import { graphql } from "react-relay"
+import { ViewingRoomStatementRoute_Test_QueryRawResponse } from "__generated__/ViewingRoomStatementRoute_Test_Query.graphql"
+import { Breakpoint } from "@artsy/palette"
+import { ViewingRoomStatementRouteFragmentContainer } from "../ViewingRoomStatementRoute"
+import { ViewingRoomStatmentRouteFixture } from "Apps/ViewingRoom/__tests__/Fixtures/ViewingRoomStatementRouteFixture"
```
Comment (sweir27) [inline src/Apps/ViewingRoom/Routes/Statement/__tests__/ViewingRoomStatementRoute.test.tsx:7]: I tend to prefer putting once-used fixtures in the file in which they're used. Makes it a little longer but means we don't need to `export` and risk keeping old fixture files around that aren't used.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.36.13`](https://github.com/artsy/reaction/releases/tag/v26.36.13) :rocket:

File: src/Apps/ViewingRoom/Routes/Statement/__tests__/ViewingRoomStatementRoute.test.tsx (hunk 2 above)
Comment (damassi) [reply src/Apps/ViewingRoom/Routes/Statement/__tests__/ViewingRoomStatementRoute.test.tsx:7]: Thats a good point, yeah right now things are all spread out. Most `Apps/*` fixtures actually ended up `Apps/__tests__/Fixtures/*`, kinda far away, but it makes sense to colocate everything. Will update in follow-up.

**Comment by damassi** [comment]
Yeah, unfortunately we're a long way from that world unless we start breaking up Reaction and paying overtime :/

**Pull request merged**